.github/workflows/ci.yml

```diff
@@ -46,6 +46,8 @@ jobs:
     if: github.ref == 'refs/heads/main'
     runs-on: ubuntu-latest
     timeout-minutes: 5
+    permissions:
+      contents: write
```

Comment:
Does this need read permission for the repo checkout?

Reply:
It will already have read permissions, I believe this is additional permissions (this is similar to changes required in another job that had this issue in Embed)